tools/find_image.rb

```diff
@@ -33,14 +33,14 @@ def run
           :url => "https://amd64.ocp.releases.ci.openshift.org/"
         raise "missing query string" if options.query.nil?
         print("Querying #{options.url} against #{options.build_type}\n")
-        doc = Nokogiri::HTML(URI.open(options.url).read)
+        doc = Nokogiri::HTML(URI(options.url).open.read)
```

Comment:
is not `URI.open` calling `URI.open` directly? I don't mind the change if it works with Ruby 2.4 but I can't understand where the issue comes from.

Reply:
@akostadinov it works with ruby `2.4`
```
$ ./tools/find_image.rb -q 1937238
Querying https://amd64.ocp.releases.ci.openshift.org/ against nightly-
Pattern '1937238' found in payload: https://amd64.ocp.releases.ci.openshift.org//releasestream/4.7.0-0.nightly/release/4.7.0-0.nightly-2021-03-13-033953
pruan@cube ~/workspace/bushslicer (fix_URI_open)$ ruby -v
ruby 2.4.10p364 (2020-03-31 revision 67879) [x86_64-linux]
```

Reply:
@akostadinov let me know if you want me to close this one since it seems to affecting on Mac